src/olympia/devhub/templates/devhub/addons/edit/describe.html

```diff
@@ -73,7 +73,7 @@ <h3>
                 {{ _("Description") }}
                 {{ tip(None,
                        _("A longer explanation of features, functionality, and other relevant information. "
-                         "This field is only displayed on the add-on's listing. " +
+                         "This field is only displayed on the product's page. " +
```

Comment:
This should probably be "This field is only displayed on the add-on's product page" since we do not reference Add-ons as products.

cc @jvillalobos to confirm though

Reply:
Yes, "add-on's product page" is how we call it now.

Reply:
@EnTeQuAk @jvillalobos Thank you for the review. I will update my PR with the changes in sometime 👍